bucket/httpstat.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/davecheney/httpstat",
+    "description": "httpstat visualizes curl(1) statistics in a way of beauty and clarity.",
+    "version": "1.0.0",
+    "license": "MIT", 
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/davecheney/httpstat/releases/download/v1.0.0/httpstat-windows-386-v1.0.0.exe#/httpstat.exe",
+            "hash": "9B43886614830830A84FAE87CBBB2F8EF01B09B1986A3A4F4AA77978887BAEE3"
+        },
+        "64bit": {
+            "url": "https://github.com/davecheney/httpstat/releases/download/v1.0.0/httpstat-windows-amd64-v1.0.0.exe#/httpstat.exe",
+            "hash": "7CF510C8927A23DB19E0D579BF44C3C325ABE0760E34FEA20567461759B794E9"
+        }
+    },
+    "checkver": {
```

Comment:
```suggestion
    "checkver": "github",
```

Reply:
By defining checkver to "github", the checkver.ps1 is able to figure out how to fetch the latest version by looking homepage/architecture urls?

Reply:
Applied.

Reply:
Yeah, see https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#special-cases